packages/react-admin/src/reducer/admin/resource/list/ids.js

```diff
@@ -70,6 +71,19 @@ export default resource => (
 
             return newState;
         }
+        case CRUD_DELETE_MANY_SUCCESS: {
+            const newState = previousState.filter(
```

Comment:
Same issue here, if a individual `DELETE` fails the state would become out of sync. Using the `requestPayload` ids is a bad idea imo. Also, the data state isn't cleared (this can be delayed for later implementation). 
And what about `CRUD_UPDATE_MANY_SUCCESS`? should the state be updated? or are you relying on `refresh`?

Reply:
Yes, we rely on refresh. Also there is no individual `DELETE`, we introduced `DELETE_MANY` and `UPDATE_MANY`. Error handling of individual requests if that's how the dataProvider handle those types, should be handled by the dataProvider

Reply:
Ok, so it's considered an atomic operation. It would be nice to add a parameter to the `dataActions` to disable the default `showNotification` on failure. This allows me to implement a custom saga on failure and not showing the notification.

Reply:
Agreed but out of this PR scope IMO